tools/devtool

```diff
@@ -1283,6 +1283,15 @@ cmd_shell() {
     return $ret
 }
 
+cmd_sh() {
```

Comment:
there used to be a user-facing command in other PRs...

Reply:
oh the help... yeah where did it go? let me fix.

Reply:
oh the help... yeah where did it go? let me fix.